### PR TITLE
fix: python sdk config example

### PR DESCRIPTION
**Pull request by DanielNwaeze**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Review by vinckr — COMMENTED** [review]
Thanks @DanielNwaeze 
this looks great! 
Can you please sign the CLA as well, then I do a final review and merge. 

Thanks again 🙌

**Comment by DanielNwaeze** [comment]
I have signed the CLA, thanks @vinckr

File: docs/kratos/sdk/10_python.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+Then import the package:
+```python
+import ory_client
+```
+
+## **CheckPermission**
+As an example, let's check for whether a given subject has the necessary permissions.
+### Example
+
+* Bearer Authentication (oryAccessToken):
```
Comment (vinckr) [inline docs/kratos/sdk/10_python.mdx:70]:
```suggestion
## CheckPermission

As an example, let's check for whether a given subject has the necessary permissions.

### Example

- Bearer Authentication (oryAccessToken):
```

Just some formatting issues.

**Review by vinckr — CHANGES_REQUESTED** [review]
Thanks! 
2 small things: 

- Can you move the document to docs/keto/sdk/ please, since the example is for permissions
- minor formatting issue I marked below

**Comment by DanielNwaeze** [comment]
@vinckr  i have made the necessary changes

**Review by vinckr — APPROVED** [review]

**Comment by vinckr** [comment]
awesome thanks so much for the contribution @DanielNwaeze 
I will fix the format and sidebar in a followup PR, bc that is always a bit annoying to do when you do not have write access here.

**Comment by DanielNwaeze** [comment]
No problem, it's fine. Thank you for the merge @vinckr

**Pull request merged**